01/fuzzygwalchmei/wordvalue.py

```diff
@@ -0,0 +1,20 @@
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words():
+    """Load dictionary into a list and return list"""
+    return [line.rstrip('\n') for line in open(DICTIONARY)]
+
+def calc_word_value(word):
+    """Calculate the value of the word entered into function
+    using imported constant mapping LETTER_SCORES"""
+    return sum(LETTER_SCORES.get(letter.upper(), 0) for letter in word)
+
+def max_word_value(words=None):
+    """Calculate the word with the max value, can receive a list
+    of words as arg, if none provided uses default DICTIONARY"""
+    if words is None:
+        words = load_words()
+    return max(words, key=calc_word_value)
```

Comment:
ha this is nice....I always tell the people to build a list of tuples (word, value) and then use `max` with key argument but never thought passing a function...this is so nice and clean great